### PR TITLE
Put and Get Bucket Notification Using Tenant User

### PR DESCRIPTION
**Pull request by anrao19**
Put and Get Bucket Notification Using Tenant User
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83575500
LOG: 
single-site: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/ss_test_bucket_notification_with_tenant_user.console.log
multi-site: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/ms_test_bucket_notification_with_tenant_user.console.log

does not effect existing TC: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/test_bucket_notification_kafka_none_persistent_delete.console.log

### CODE REVIEW

**Review by viduship — COMMENTED** [review]
@anrao19, This looks good to me .

Please also include tenancy at multisite. You may create a symlink of this to the multisite_configs/. You may have it in the next PR if not this one.

**Review by viduship — DISMISSED** [review]

**Comment by anrao19** [comment]
> @anrao19, This looks good to me .
> 
> Please also include tenancy at multisite. You may create a symlink of this to the multisite_configs/. You may have it in the next PR if not this one.

tseted and include in same pr

**Review by viduship — APPROVED** [review]

File: rgw/v2/tests/s3_swift/test_bucket_notifications.py [hunk 1]
```diff
@@ -236,16 +254,22 @@ def test_exec(config, ssh_con):
...
     reusable.check_sync_status()
 
     # check for any crashes during the execution
     crash_info = reusable.check_for_crash()
-    if crash_info:
-        raise TestExecError("ceph daemon crash found!")
+    if config.user_remove:
+        if crash_info:
+            raise TestExecError("ceph daemon crash found!")
+        for i in all_users_info:
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_bucket_notifications.py:268]: @anrao19 , could we have a check on user remove after checking the crash [ at line 268]

**Review by ckulal — COMMENTED** [review]
Looks Good

**Review by hmaheswa — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**